iot/dotnet-iot-and-nanoframework/dotnet-iot/Program.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
```

Comment:
It would be great to add a MIT style header, in this case, it's clear that the code can be reused for any purpose. Same for other source code files.

Reply:
Thanks this is added as well.